.github/workflows/development_infrastructure.yml

```diff
@@ -757,6 +786,64 @@ jobs:
     #       fi
   # END Redis setup
 
+  # START ClamAV setup
+  clamav:
+    name: ClamAV Setup
+    needs: [set-environment, vnet, private-dns]
+    environment:
+      name: ${{ needs.set-environment.outputs.environment }}
+    runs-on: [self-hosted, linux, deployment]
+    steps:
+      - uses: azure/login@v1
+        with:
+          creds: ${{ env.credentials }}
+
+      - name: Defaults
+        uses: azure/cli@v1.0.0
+        with:
+          inlineScript: |
+            az configure --defaults location=uksouth
+            az configure --defaults group=${{secrets.RESOURCE_GROUP }}
+
+      # Need to set up log-analytics and use existing workspace resource here for managed environment and application insights
+      # - name: Workspace Setup
+      #   uses: azure/cli@v1.0.0
+      #   with:
+      #     inlineScript: |
+      #       az monitor log-analytics workspace create --workspace-name ${{ secrets.RESOURCE_GROUP}}-Logs-Workspace
```

Comment:
Lots of commented out code in the file, just wanted to double check it all needs to be here